src/libstd/process.rs

```diff
@@ -552,6 +552,15 @@ impl Command {
 
     /// Configuration for the child process's standard input (stdin) handle.
     ///
+    /// Defaults to [`inherit`] when used with `spawn` or `status`, and
+    /// defaults to [`piped`] when used with `output`.
+    ///
+    /// On Windows, if the `#![windows_subsystem = "windows"]` attribute is
```

Comment:
can you put the windows-specific lines below a 'Platform-specific behavior' header like we do [here](https://github.com/rust-lang/rust/blob/5f578dfad0dd5d43b28eff71a7e857d10c3f55fe/src/libstd/fs.rs#L1223-L1227)?